### ISSUE TITLE
how to compare the new plot with an old plot?

### ISSUE THREAD

**Issue by isalirezag** [facebookresearch/visdom#429]
is there a way if i have a list of my losses for my training case 1(that already is trained) and want to compare it with my new training network case 2 which is training

**Comment by JackUrb** [comment]
Hi @isalirezag, you should be able to plot the existing loss as one trace, and then append a new trace to the same plot by using the same `win=win_id` param. There are examples of appending to a new trace in the demo at `examples/demo.py`

**Comment by isalirezag** [comment]
I am a newbie to visdom
let say im plotting somthing as follows:

```

from visdom import Visdom
import numpy as np
import torch

viz = Visdom()

def update_vis_plot(iteration, loc, conf, window1, window2, update_type,
                    epoch_size=1):
    viz.line(
            X=torch.ones((1, 3)).cpu() * iteration,
            Y=torch.Tensor([loc, conf, loc + conf]).unsqueeze(0).cpu() / epoch_size,
            win=window1,
            update=update_type
            )
    # initialize epoch plot on first iteration
    if iteration == 0:
        viz.line(
                X=torch.zeros((1, 3)).cpu(),
                Y=torch.Tensor([loc, conf, loc + conf]).unsqueeze(0).cpu(),
                win=window2,
                update=True
                )
        

def create_vis_plot(_xlabel, _ylabel, _title, _legend):
    
    return viz.line(
            X=torch.zeros((1,)).cpu(),
            Y=torch.zeros((1, 3)).cpu(),
            opts=dict(
                    xlabel=_xlabel,
                    ylabel=_ylabel,
                    title=_title,
                    legend=_legend
                    )
            )
            

vis_title = 'SSD.PyTorch on ' + 'Alireza'
vis_legend = ['Loc Loss', 'Conf Loss', 'Total Loss']
iter_plot = create_vis_plot('Iteration', 'Loss', vis_title, vis_legend)
epoch_plot = create_vis_plot('Epoch', 'Loss', vis_title, vis_legend)


for iteration in range(1000):
    
    if  iteration != 0:
        epoch = iteration * 2
        loc_loss = iteration * 3
        conf_loss = iteration * 4
        
        WIN = update_vis_plot(epoch, loc_loss, conf_loss, epoch_plot, None,
                        'append', epoch_size)
```
Now I want to plot another one, for example:

```
for iteration in range(1000):
    
    if  iteration != 0:
        epoch = iteration * 2
        loc_loss = iteration * 2
        conf_loss = iteration * 2
        
        WIN = update_vis_plot(epoch, loc_loss, conf_loss, epoch_plot, None,
                        'append', epoch_size)
            
```
one the top of the previous one.
How should i do that?

**Comment by JackUrb** [comment]
You'll need to set a trace name if you want to append a new trace on top, as in the demo here: https://github.com/facebookresearch/visdom/blob/9dba3f645b341e6cdb4895cb40fddd4c0227e5d2/example/demo.py#L178

**Comment by JackUrb** [comment]
@isalirezag did you end up getting this to work?
